Published-source: https://deb.freexian.com/extended-lts/pool/main/p/php7.0/php7.0_7.0.33-0+deb9u26.debian.tar.xz
Published-patch: 0078-fix-test-output.patch
Scope: selected PHP runtime components and their tests; packaging and unrelated extensions omitted.

From: Remi Collet <remi@remirepo.net>
Date: Tue, 28 May 2019 07:28:46 +0200
Subject: fix test output

---
 ext/iconv/tests/bug78069.phpt | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

--- a/ext/iconv/tests/bug78069.phpt
+++ b/ext/iconv/tests/bug78069.phpt
@@ -10,6 +10,7 @@ $hdr = iconv_mime_decode_headers(file_get_contents(__DIR__ . "/bug78069.data"),2
 var_dump(count($hdr));
 ?>
 DONE
---EXPECT--
+--EXPECTF--
+Notice: iconv_mime_decode_headers%s
 int(1)
-DONE
\ No newline at end of file
+DONE
